zerver/views/reactions.py

```diff
@@ -66,3 +67,60 @@ def remove_reaction_backend(request, user_profile, message_id, emoji_name):
     do_remove_reaction_legacy(user_profile, message, emoji_name)
 
     return json_success()
+
+@has_request_variables
+def add_reaction(request, user_profile, message_id,
+                 emoji_name=REQ(default=None),
+                 emoji_code=REQ(default=None),
+                 reaction_type=REQ(default="unicode_emoji")):
+    # type: (HttpRequest, UserProfile, int, Text, Text, Text) -> HttpResponse
+    message, user_message = access_message(user_profile, message_id)
+
+    query = Reaction.objects.filter(user_profile=user_profile,
+                                    message=message,
+                                    emoji_code=emoji_code,
+                                    reaction_type=reaction_type)
+    if query.exists():
```

Comment:
We don't need to create the `query` variable here; seems a bit cleaner without it.